node-bb/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:lts
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+ARG NODE_ENV
+ENV NODE_ENV $NODE_ENV
+
+COPY install/package.json /usr/src/app/package.json
+
+RUN npm install --only=prod && \
+    npm cache clean --force
+    
+COPY . /usr/src/app
+
+ENV NODE_ENV=production \
+    daemon=false \
+    silent=false
+
+EXPOSE 4567
+
+CMD node ./nodebb build ;  node ./nodebb start
```

Comment:
I'm not 100% sure if "nodebb start" (works without specifying `node` because of shebang) works as expected inside docker-
IIRC `./nodebb start` just spins up a daemon service and terminates itself soon after.
The current systemd script runs `/usr/bin/env node loader.js`, which spins up the process directly.

Reply:
This docker file is verbatim from: https://hub.docker.com/r/nodebb/docker/

Your guess is as good as mine if there is anything missing.